src/test/java/com/google/api/codegen/testdata/ruby_doc_main_library.baseline

```diff
@@ -252,7 +252,7 @@ module Library
 
         google_api_client = "gl-ruby/#{RUBY_VERSION}"
         google_api_client << " #{lib_name}/#{lib_version}" if lib_name
-        google_api_client << " gapic/ gax/#{Google::Gax::VERSION}"
+        google_api_client << " gapic/0.1.0 gax/#{Google::Gax::VERSION}"
```

Comment:
@landrito Is this the version of the code generator or the generated code? We want the latter.